README.md

```diff
@@ -71,6 +73,7 @@ Each expression is an array table which has three elements:
 
 |operator|description|example|
 |--------|-----------|-------|
+|!       |reverse the result|{"arg_name", "!", "~~", "[a-z]+"}|
```

Comment:
This operator is not commonly used and can be put last.

Reply:
@membphis 
Updated.